README.md

```diff
@@ -52,6 +52,8 @@ Will call `callback` with an array of detailed ad objects.
 
 Values for `locationId` and `categoryId` can be found by performing a search and looking at the POST request parameters or the URL Kijiji redirects to. For example, after setting the location to Ottawa and selecting the "cars & vehicles" category, Kijiji redirects to http://www.kijiji.ca/b-cars-vehicles/ottawa/c27l1700185. The last part of the URL (c27l1700185) is formatted as c[categoryId]l[locationId]. So in this case, `categoryId` is 27 and `locationId` is 1700185.
 
+By default, each query result is scraped. To suppress this behavior and return only the data retrieved by the initial query, set the `scrape` preference to `false`.
```

Comment:
Fair point. I'm just trying to keep the readme concise. How about something like this (if more optional properties are added in the future the prefs example could be separated into two required and optional examples)

* `prefs` - Contains Kijiji ad search category and location:
 ```js
{
     "locationId": <Kijiji location id>,
     "categoryId": <Kijiji ad category id>,
     "scrapeInnerAd": true/false (default true)
}
```